content/docs/import/import-from-neon.md

```diff
@@ -12,7 +12,7 @@ This guide describes how to migrate a database from one Neon project to another
 ## Important considerations
 
 - **Upgrading the Postgres version**: When upgrading to a new version of Postgres, always test thoroughly before migrating your production systems or applications. We also recommend familiarizing yourself with the changes in the new version of Postgres, especially those affecting compatibility. For information about those changes, please refer to the official Postgres [Release 15](https://www.postgresql.org/docs/release/15.0/) or [Release 16](https://www.postgresql.org/docs/16/release-16.html) documentation.
-- **Piping considerations**: Piping is not recommended for medium (> 1 GB) and large databases (> 5 GB), as it is susceptible to failures during lengthy migration operations (see [Pipe pg_dump to pg_restore](/docs/import/import-from-postgres#pipe-pgdump-to-pgrestore) for more information). If your database is larger than 1 GB, we recommend performing the dump and restore as separate operations. For instructions, see [Import data from Postgres](/docs/import/import-from-postgres).
+- **Piping considerations**: Piping is not recommended for medium (> 1 GiB) and large databases (> 5 GiB), as it is susceptible to failures during lengthy migration operations (see [Pipe pg_dump to pg_restore](/docs/import/import-from-postgres#pipe-pgdump-to-pgrestore) for more information). If your database is larger than 1 GiB, we recommend performing the dump and restore as separate operations. For instructions, see [Import data from Postgres](/docs/import/import-from-postgres).
```

Comment:
FWIW i don't think it's critical to have GiB here.
up to docs team to decide

Reply:
This is true. No critical here.

Reply:
@danieltprice what is our preference? GiB or GB when referring to sizes outside of Neon?

Reply:
OK. Having given this some more thought, I would recommend GB unless the context requires precision, like size calculation instructions. Not everyone is familiar with GiB. Even in those instances, we might use GB, and add a note about GB being "binary gigabytes".